{"name": "test_register_page", "status": "failed", "statusDetails": {"message": "AssertionError: The user is not logged in successfully.\nassert False\n +  where False = user_is_logged()\n +    where user_is_logged = <pages.actions.register_actions.RegisterActions object at 0x0000022D645321B0>.user_is_logged", "trace": "venv\\Lib\\site-packages\\_pytest\\fixtures.py:898: in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\nfeatures\\steps\\test_register.py:31: in step_registered_user_successfully\n    assert register_page.user_is_logged(), \"The user is not logged in successfully.\"\nE   AssertionError: The user is not logged in successfully.\nE   assert False\nE    +  where False = user_is_logged()\nE    +    where user_is_logged = <pages.actions.register_actions.RegisterActions object at 0x0000022D645321B0>.user_is_logged"}, "description": "register.feature: Register User", "start": 1729969949351, "stop": 1729969952908, "uuid": "40d321a9-3a2c-4dd8-b4cf-5f883fa8f1c3", "historyId": "356fda2fa29605565c7e3b758b79507d", "testCaseId": "356fda2fa29605565c7e3b758b79507d", "fullName": "features.steps.test_register#test_register_page", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures"}, {"name": "parentSuite", "value": "features.steps"}, {"name": "suite", "value": "test_register"}, {"name": "host", "value": "manuel"}, {"name": "thread", "value": "13432-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "features.steps.test_register"}]}